doc/field_guide__eclipsing.md

```diff
@@ -0,0 +1,32 @@
+## Eclipsing variables (ecl)
+
+Eclipsing variables are binary stars systems in which the orbital plane lies nearly in the line of sight of the observer. The binary components pass in front of one another from observer's perspective and so their combined brightness decreases periodically.
+
+### Classification and numbers
+- Supertypes
+  - variable
+  - periodic
+- Subtypes
+  - EA
+  - EB
+  - EW
+- Occurrence rate: very common, about 10<sup>6</sup> expected in ZTF data
+
+### ZTF light curves
+![ZTF eclipsing](data/eclipsing.png)
+
+#### Description
+Eclipsing variables show regular, periodic "V-shape" dips in their phase folded light curves, which may be sharp or rounded, depending on the system. Their periods may range from minutes to years. Eclipse amplitudes vary from ~0.1 to several magnitudes.
+
+#### Light curve characteristics
+- periodic variable
+- Range of amplitudes (~0.1 to >1 mag)
+- Periods (from minutes to years)
+
+![HR diagram of eclipsing variables](data/hr__eclipsing.png)
+
+![Period histogram of eclipsing variables](data/period__eclipsing.png)
+
+### References and further reading:
+- Graczyk et al. (2011) <a href="https://ui.adsabs.harvard.edu/abs/2011AcA....61..103G/abstract">The Optical Gravitational Lensing Experiment. The OGLE-III Catalog of Variable Stars. XII. Eclipsing Binary Stars in the Large Magellanic Cloud</a>

```

Comment:
We were writing out the whole citation I think, i.e. including journal and so on.